警告対応 : 戻り値が無い

# 目的 : Must

期待通りの振る舞いをさせる
## 背景

makeで以下の警告が出ている

``` shell
../workspace/hackEV/ArmModule.cpp: In function 'ER initialize_arm_position()':
../workspace/hackEV/ArmModule.cpp:122:1: warning: no return statement in function returning non-void [-Wreturn-t    ype]
 }
```
# やる事 : Must

背景に記載した警告を消す
# 考慮した事
## [リスク](https://kotobank.jp/word/%E3%83%AA%E3%82%B9%E3%82%AF-183705)

なし
## [懸念事項](https://kotobank.jp/word/%E6%87%B8%E5%BF%B5-490895)

なし

--- a/hrp2/sdk/workspace/hackEV/ArmModule.cpp
+++ b/hrp2/sdk/workspace/hackEV/ArmModule.cpp
@@ -119,6 +119,7 @@ ER initialize_arm_position()
         ev3_speaker_play_tone(NOTE_A6, 100);
     }
 
+    return  result;
 }
 
 /**

